paperqa/rate_limiter.py

```diff
@@ -301,7 +310,7 @@ async def try_acquire(
         """Returns when the limit is satisfied for the namespace_and_key.
 
         Args:
-            namespace_and_key (:obj:`tuple[str, str | MatchAllInputs]`): is
+            namespace_and_key (:obj:`tuple[str, str]`): is
```

Comment:
What is this `:obj:` syntax here? Normally it's good to not restate types if they're in the docstring already

Reply:
it's in the style guide for the google doc strings, I think for sphinx compatibility

Reply:
Fwiw, either Sphinx autodoc or napoleon (I can't remember which) will pull in type hints if they are there

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods note that it does not restate type hints in `Args` or `Returns`

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html in this one, note there's not type hints at all

Anyways, moral of the story I have seen it where people later update or expand type hints, but then don't update the repeated version in the docstring `Args` or `Returns`